### PR TITLE
chore(containers): Add minio container

### PR DESCRIPTION
**Pull request by xmnlab**
This PR aims to add a container for minio that would allow a data storage across all the services

### CODE REVIEW

**Comment by xmnlab** [comment]
PS: it is still missing tests

**Review by fccoelho — APPROVED** [review]
Looks good to me.

File: containers/compose-base.yaml [hunk 1]
```diff
@@ -126,5 +127,30 @@ services:
...
+      "--console-address",
+      ":9090"
+    ]
+    ports:
+      - ${MINIO_PORT_1}:9000
+      - ${MINIO_PORT_2}:9090
+    volumes:
+      - minio:/data
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
```
Comment (fccoelho) [inline containers/compose-base.yaml:147]: We should open an issue for this health check endpoint to be added to the status check airflow DAG

File: README.md [hunk 2]
```diff
@@ -222,6 +222,20 @@ Create a **FERNET KEY**:
...
+
+```bash
+make dev-create-s3-credentials
+```
+
+Note: it needs the AWS env variable on the environment (or from .env file).
+
 ## Deployment
 
 The deployment is executed by **Ansible** and triggered by **GitHub Actions**.
```
Comment (fccoelho) [inline README.md:241]: Maybe we should add a section on how to use the service, here in the README

**Review by fccoelho — APPROVED** [review]
Looks Good!

File: containers/compose-base.yaml (hunk 1 above)
Comment (xmnlab) [reply containers/compose-base.yaml:147]: done

**Pull request merged**